### ISSUE TITLE
Importing concerto files from v5.0.beta.2.160 to v5.0.beta.2.186 Error

### ISSUE THREAD

**Issue by roseKQ** [campsych/concerto-platform#110]
Hi,

When I try and import concerto files from v5.0.beta.2.160 to v5.0.beta.2.186 I get the following error 

![concerto_error](https://cloud.githubusercontent.com/assets/11443407/24403908/e83c25bc-13b6-11e7-9216-5e6977f62192.PNG)

Please advise, thanks

**Comment by przemyslawlis** [comment]
Hi,
Just as the message says. There was a export file format change in one of the update, so any older export files won't work. 
You could update your Concerto Platform where the content is located to the latest version and reexport it from there.

**Comment by rumiexile** [comment]
Hi przemyslawlis, 

Do you have a guide on how to update Concerto on Amazon AWS?

**Comment by postview** [comment]
Facing similar issue but don't have access to the older version of the platform (was on an AWS instance which I shutdown)

Tried to import  test from version"5.0.beta.2.186" to V5.0.beta13 and facing issue.

Requesting if someone can assist in getting the code for version "5.0.beta.2.186" .
The experiment I created was some 2.5 years ago. Need to replicate the experiment and don't remember all the steps in the experiment. @przemyslawlis
